src/record.py

```diff
@@ -0,0 +1,44 @@
+"""Record class."""
+from parse import get_parser, get_card
+from extract import read_to_dict
+from standardize import standardizer, strip_payment
+from common import flip_spending_sign
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+class CardRecord(object):
+    def __init__(self, filename):
+        self.filename = filename
+        self.card = get_card(filename)
```

Comment:
do we use this anywhere? Also each card record has a `source` field which is the card it came from so the information is also available row by row across all cards.

Reply:
I'd like to use this in parsers down the line instead of parsing the filename, since when you download records from the website they don't come formatted as amex_*.CSV etc. So being able to assign card type independently of what a file is named seems like a useful property.

Reply:
To expand on that, in the next line, I would change it to 
```
self.parser = get_parser(self.card)
```